### ISSUE TITLE
Name is not defined for all the info-tooltips present on the page.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#88013]
### What happened?

Name & role is not defined for ‘More info’ control, due to which screen reader is just narrating as ‘Group’ for it.  

https://github.com/grafana/grafana/assets/93735775/de2d7557-5d82-472b-a7df-b4a4b4b893d8

### What did you expect to happen?

>Appropriate name should be defined for the control such as ‘More information about xxxxxxx’. 

>Role should be defined as ‘Button’. 

>Help text present for ‘More info’ control should be visible only on pressing enter & screen reader should also narrate the same on pressing enter. 

### Did this work before?

NA

### How do we reproduce it?

1.Open the URL [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1) in edge dev browser. 

2.’Grafana Play Home-Dashboards Grafana’ page will open. 

3.Navigate to ‘Home’ link & select it. 

4. Navigate to ‘Alerting’ link present in ‘Feature show cases’ section & activate it. 

5.’Alerting’ page will open. 

6.Turn on Narrator using ‘Ctrl+Windows+Enter’ keys. 

7.Navigate to ‘More info’ (i) button via tab key & observe the screen reader narration

### Environment (with versions)?

Grafana: v11.1.0-69051 (9d44c8e8cf) 
OS: Version 24H2 (OS Build 26100.1) 
Browser: Version 125.0.2506.0 (Official build) dev (64-bit) 

### Grafana platform?

None

**Comment by L-M-K-B** [comment]
I could reproduce the issue and after a short investigation I found that the issue is based in this component: packages/grafana-ui/src/components/PanelChrome/PanelDescription.tsx

**Comment by ashharrison90** [comment]
yep, this specific to `PanelDescription` and it's interaction with `TitleItem`. briefly investigated, and found that swapping the order of `Tooltip` and `TitleItem` fixes the a11y problem (it correctly announces the tooltip text via voiceover), but causes the styling to change somewhat.

i think that's mostly how we'd fix it though, the tooltip needs to wrap the triggering element directly so it can correctly set `aria-describedby`

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by sornapudisuresh** [comment]
I'd like to take this.

Plan: fix `PanelDescription` / `TitleItem` so the tooltip wraps the trigger directly (per @ashharrison90's note), giving the control a proper accessible name/role for screen readers while preserving styling as closely as possible.

Please assign me if that works.
